firmware/ec/src/devices/ltc4275.c

```diff
@@ -263,6 +263,6 @@ void ltc4275_update_status(const LTC4275_Dev *dev)
     } else {
         PDStatus_Info.state = LTC4275_STATE_NOTOK;
         PDStatus_Info.pdalert = LTC4275_DISCONNECT_ALERT;
-        PDStatus_Info.pdStatus.classStatus == LTC4275_CLASSTYPE_UNKOWN;
+        PDStatus_Info.pdStatus.classStatus = LTC4275_CLASSTYPE_UNKOWN;
```

Comment:
Lucky that classStatus is uint8

Reply:
Yeah, that should really not be uint8_t in the definition, but instead ePDState....Fix in later PR as -fshort-enums makes it work in unit tests right now.